src/App.css

```diff
@@ -20,3 +20,16 @@
 .App-link {
   color: #09d3ac;
 }
+
```

Comment:
Hey, I really like your UI but even while making it fancy, let's stick to the requirements of the milestone like using making the width 700px, the display and button 100px, the wide button 25%.
- The part where you used other colours is fine i guess the important thing there is using default colour and passing props.
BUT:
- This whole milestone is about following a given design style, if we don't do that then there is no point of it existing.